docs/sql/casts.md

```diff
@@ -27,4 +27,10 @@ range values.
 Conversions from decimal and floating point types to integer types
 always truncate the decimal digits (round towards zero).  For example,
 `CAST(2.9 AS INTEGER)` returns 2, while `CAST(-2.9 AS INTEGER)`
-returns -2.
\ No newline at end of file
+returns -2.
+
+Casts of strings to numeric types produce the result zero (0) when the
```

Comment:
If this is standard SQL or so, give a reference.

Reply:
You are funny. The sql standards are all paywalled. And no one implemented the standard anyway.

Reply:
Are other DBs publicly documented to behave this way? Postgres or MySQL for example?

Reply:
In fact the standard says clearly that in this case one should trigger a runtime error. Not null, not zero.
I will check if it's a Calcite bug or a "feature".

Reply:
I will change the documentation for this case and file an issue.